readme.md

```diff
@@ -134,6 +135,32 @@ Directories are recursive by default. Directories named `fixtures` and `helpers`
 
 When using `npm test`, you can pass positional arguments directly `npm test test2.js`, but flags needs to be passed like `npm test -- --verbose`.
 
+### The `--match` flag
+
+The `--match` flag allows you to run a subset of the tests in your suite that have a matching title. This is achieved with simple wildcard patterns. For more information, check out [sindresorhus/matcher](https://github.com/sindresorhus/matcher).
+
+```
+# match titles ending with 'foo'
+$ ava --match=*foo
+
+# match titles starting with 'foo'
+$ ava --match=foo*
+
+# match titles containing 'foo'
+$ ava --match=*foo*
+
+# match titles not containing 'foo'
+$ ava --match=!*foo*
```

Comment:
Me being the inconsiderate Bash user, I opted for no quotes. I'll single quote 'em all :)